plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/AwsSecurityConfig.java

```diff
@@ -66,4 +69,33 @@ public AwsSecurityConfig setRegion(String region)
         this.region = region;
         return this;
     }
+
+    @NotNull
+    public Optional<String> getIamRole()
+    {
+        return Optional.ofNullable(iamRole);
+    }
+
+    @Config("elasticsearch.aws.iam-role")
+    @ConfigDescription("Optional AWS IAM role to assume for authenticating. If set this role will be used to get credentials and sign requests." +
```

Comment:
Missing space after `.` Can we make it shorter?

Reply:
Applied.